src/servers/http_server.cc

```diff
@@ -1231,26 +1231,66 @@ void
 HTTPAPIServer::HandleRepositoryIndex(
     evhtp_request_t* req, const std::string& repository_name)
 {
-  if (req->method != htp_method_GET) {
+  if (req->method != htp_method_POST) {
```

Comment:
Don't the clients need to modified as well?

Reply:
Yes, updated